compiler/rustc_codegen_llvm/src/debuginfo/mod.rs

```diff
@@ -61,7 +61,7 @@ pub struct CodegenUnitDebugContext<'ll, 'tcx> {
     llcontext: &'ll llvm::Context,
     llmod: &'ll llvm::Module,
     builder: &'ll mut DIBuilder<'ll>,
-    created_files: RefCell<FxHashMap<Option<(u128, SourceFileHash)>, &'ll DIFile>>,
+    created_files: RefCell<FxHashMap<Option<([u8; 16], SourceFileHash)>, &'ll DIFile>>,
```

Comment:
Can this use `Fingerprint`?

Reply:
`Fingerprint` itself has no documentation at all, and individual users document their usage of it. And its interface is so flexible and supports no encapsulation, so it seems to me like it's treated more like a bag of utilities than a type.

I'm trying to avoid using it anywhere else.